packages/@coorpacademy-components/src/atom/html/index.native.tsx

```diff
@@ -111,7 +112,30 @@ const Html = (props: Props) => {
     [numberOfLines]
   );
 
+  const FontRenderer = useCallback(
+    (htmlAttribs: CustomRendererProps<TBlock>, _children: string) => {
+      // @ts-expect-error ts(2339) --> color may actually exist on <font>
+      if (htmlAttribs.color) {
+        disableBaseFontStyleColor(true);
+      }
+      return (
+        <Text
+          key={1}
+          style={{
+            ...baseFontStyle,
+            // @ts-expect-error ts(2339) --> color is a string
+            color: htmlAttribs.color?.replace(/ /g, '')
```

Comment:
isn't it possible to use a "as" instead of // @ts-expect-error ts(2339) --> color is a string ?

Reply:
the (false) error is 
```
"Property 'color' does not exist on type 'CustomRendererProps<TBlock>'."
```